core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -185,6 +196,20 @@ export class CreateServiceComponent implements OnInit {
 
   }
 
+  //function to validate event source names
+  validateEvents(value){
+
+    if(value != null &&(value[0] === '-' || value[value.length - 1] === '-')){
```

Comment:
cant you combine all these ifs into one?

Reply:
done